src/components/PoolDashboard/Modals/StakeDaiModal.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Modal from '@material-ui/core/Modal';
+import { Typography } from '@material-ui/core';
+import ErrorIcon from '@material-ui/icons/Error';
+
+import MainButton from '../../UI/MainButton';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  body: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    flexDirection: 'column',
+    textAlign: 'center',
+    width: 400,
+    height: 300,
+    backgroundColor: 'white',
+    borderRadius: 20,
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3),
+    outline: 0,
+  },
+  logo: {
+    width: 90,
+    height: 90,
+  },
+}));
+
+const StakeDaiModal = (props) => {
+  const classes = useStyles();
+  console.log('user balance:', props.userBalance);
+  console.log('lockValue:', props.lockValue);
+
+  const modalBody = (
+    <div className={classes.body}>
+      {props.userBalance < props.lockValue ? (
```

Comment:
We should compare amounts in wei (using bignumber.js for example). That way it might be not correct. But I don't think we'll change this now

Reply:
so... leave it as it is?